docs/Using Price Feeds/l2-sequencer-feeds/l2-sequencer-flag.md

```diff
@@ -0,0 +1,157 @@
+
+### Handling outages on Optimism and Metis
+
+If the sequencer is down, messages cannot be transmitted from L1 to L2 and **no L2 transactions are executed**. Instead, messages are enqueued in the `CanonicalTransactionChain` on L1 and only processed in the order they arrived later when the sequencer comes back up. As long as the message from the validator on L1 is already enqueued in the `CTC`, the flag on the sequencer uptime feed on L2 will be guaranteed to be flipped prior to any subsequent transactions. The transaction that flips the flag on the uptime feed will be executed before transactions that were enqueued after it. This is further explained in the diagrams below.
+
+When the Sequencer is down, all L2 transactions sent from the L1 network wait in the pending queue.
+
+1. **TX3** contains Chainlink’s transaction to set the status of the sequencer as being down on L2.
+1. **TX4** is a transaction made by a consumer that is dependent on the sequencer status.
+
+![L2 Sequencer Feed Diagram](/images/data-feed/seq-down-1.png)
+
+After the sequencer comes back up, it moves moves all transactions in the pending queue to the processed queue.
+
+1. Transactions are processed in the order they arrived so **TX3** is processed before **TX4**.
+1. Because **TX3** happens before **TX4**, **TX4** will read the status of the Sequencer as being down and responds accordingly.
+
+![L2 Sequencer Feed Diagram](/images/data-feed/seq-down-2.png)
+
+### Example code for Optimism and Metis
+
+Create your consumer contract for uptime feeds similarly to contracts you would use for other Chainlink data feeds. You will need the following items:
+
+- The [sequencer uptime feed proxy address](#arbitrum): This contract is on the L2 network.
+- A contract that imports the [`AggregatorV2V3Interface.sol` contract](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/AggregatorV2V3Interface.sol): Use this interface to create a `flagsFeed` object that points to the sequencer feed proxy.
+
+You can use the sequencer uptime feed on mainnet L2 feeds, but this example uses an Ethereum Rinkeby testnet feed for development and testing purposes.
+
+```solidity Mainnet
+{% include 'samples/PriceFeeds/OptimismMetisPriceConsumer.sol' %}
+```
+
+This example includes a modified `getLatestPrice` function that reverts if `checkSequencerState` returns false. The `checkSequencerState` function reads the answer from the sequencer uptime feed and returns either a `1` or a `0`.
+
+- 0: The sequencer is up
+- 1: The sequencer is down
+
+The `startedAt` timestamp indicates when the sequencer changed status. This timestamp returns `0` if a round is invalid. When the sequencer comes back up after an outage on Optimism or Metis, wait for the `GRACE_PERIOD` to pass before accepting answers from the price data feed. Subtract `startedAt` from `block.timestamp` and revert the request if the result is less than the `GRACE_PERIOD`.
```

Comment:
Variable is named `GRACE_PERIOD_TIME` instead of `GRACE_PERIOD` in the example.

Reply:
Fixed